src/ptc/util/jms.clj

```diff
@@ -140,13 +141,26 @@
       (apply misc/shell! "gsutil" "cp" (concat sources [cloud]))
       (reduce cloudify (reduce rekey {} copy) chip-and-push))))
 
+(defn get-extended-chip-manifest
+  "Get the extended_chip_manifest_file from the JMS message"
+  [workflow]
+  (let [aou-reference-bucket (or (System/getenv "AOU_REFERENCE_BUCKET") "broad-arrays-dev-storage")
+        cloud-chip-metadata-dir (get workflow :cloudChipMetaDataDirectory)
+        bucket (first (misc/parse-gs-url cloud-chip-metadata-dir))
+        aou-chip-metadata-dir (str/replace cloud-chip-metadata-dir bucket aou-reference-bucket)
```

Comment:
I thought we need `(re-pattern bucket)` 🤔

Reply:
It can take either a string or a pattern https://clojuredocs.org/clojure.core/replace